tools/tsp-client/src/options.ts

```diff
@@ -0,0 +1,152 @@
+import { parseArgs } from "node:util";
+import { Logger, printUsage, printVersion } from "./log.js";
+import * as path from "node:path";
+import { knownLanguages, languageAliases } from "./languageSettings.js";
+import { doesFileExist } from "./network.js";
+
+export interface Options {
+  debug: boolean;
+  command: string;
+  tspConfig?: string;
+  emitter?: string;
+  mainFile?: string;
+  outputDir: string;
+  noCleanup: boolean;
+  skipSyncAndGenerate: boolean;
+  commit?: string;
+  repo?: string;
+  isUrl: boolean;
```

Comment:
What is this option `isUrl` used for?

Reply:
This is related to your comment below about requiring the commit and repo for local specs. On lines 112-129, we check if there tspconfig.yaml is in the local file system or if it is a URL. If it's a local file we need a commit sha and the repo url in order to create a valid tsp-location.yaml file. I'll use the same issue linked below to track this and change it, currently I'm just targeting matching the original Process script.